integration_tests/features/Mempool.feature

```diff
@@ -0,0 +1,83 @@
+@propagation
+Feature: Mempool
+
+ 
+  Scenario: Transactions are propagated through a network
+    Given I have 10 seed nodes
+    And I have a base node SENDER connected to all seed nodes
+    And I have 10 base nodes connected to all seed nodes
+    When I wait 100 seconds
+    When I mine a block on SENDER with coinbase CB1
+    When I mine 2 blocks on SENDER
+    When I create a transaction TX1 spending CB1 to UTX1
+    When I submit transaction TX1 to SENDER
+    Then SENDER has TX1 in its mempool inside of UNCONFIRMED pool
+    When I wait 100 seconds
+    Then TX1 is in the UNCONFIRMED pool of all nodes
```

Comment:
It is new stuff Hansie and I have started to use in the Cucumber steps but look at the `I wait` type steps in https://github.com/tari-project/tari/pull/2681 and https://github.com/tari-project/tari/pull/2686 to see a better way to do this with the time constraint so you don't have to wait a 100 second each time but wait until the condition has been met OR the max time has passed.